test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1264,6 +1283,7 @@ def revert_pbsconf(self):
             "PBS_START_MOM": None,
             "PBS_CORE_LIMIT": None,
             "PBS_SCP": None,
+            "PBS_CP": None,
```

Comment:
I am not sure we should add PBS_CP here as it is not a default key-value of pbs.conf, and this dict should have only default key-value which gets added at the time of installation. @bhroam What you think?

Reply:
Whatever you and @bhroam think is best. I have no strong opinions either way.

Reply:
It sounds like @hirenvadalia is right.  This should be for just the things in pbs.conf that are set by default when PBS starts for the first time.  PBS_CP is something that is added by the admin.

Reply:
@jdburton Seems like this is the only outstanding comment left, please address this.

Reply:
Changes have been made.